python/lsst/ts/IntegrationTests/configs/love_stress_test_configs.py

```diff
@@ -0,0 +1,62 @@
+# This file is part of ts_IntegrationTests.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import yaml
+
+from .config_registry import registry
+
+
+# Add the LOVE Stress Test script
+# configurations to the registry.
+
+registry["love_stress"] = yaml.safe_dump(
+    {
+        "location": "love.tu.lsst.org",
+        "number_of_clients": 50,
+        "number_of_messages": 5000,
+        "data": [
+            "ATAOS:0",
+            "ATCamera:0",
+            "ATDome:0",
+            "ATDomeTrajectory:0",
+            "ATMCS:0",
+            "ATHexapod:0",
+            "ATPneumatics:0",
+            "ATPtg:0",
+            "ATSpectrograph:0",
+            "MTAirCompressor:1",
+            "MTAirCompressor:2",
+            "MTAlignment:0",
```

Comment:
This needs to be changed to `LaserTracker` and I think it needs a `:1` since it's apparently an indexed component now.